src/components/announcement-dialog/AnnouncementDialog.tsx

```diff
@@ -1,6 +1,5 @@
-import { type ReactNode, forwardRef } from 'react';
+import React, { type ReactNode, forwardRef } from 'react';
```

Comment:
I thought default `React` is no longer needed for React 18 version 🤔

Reply:
I don't know. Maybe it's not. Either way, with putting the `style` in `BaseProps` we don't need the `React.CSSProperties` here, so it is removed.